distribution/docs/src/main/resources/content/_sources/csw-federation-profile-source.adoc

```diff
@@ -11,28 +11,26 @@
 
 == {title}
 
-The CSW Federation Profile Source supports the ability to search collections of descriptive information (metadata) for data, services, and related information objects.
+The CSW Federation Profile Source is ${branding}'s full fidelity CSW Federation Profile which supports the ability to search collections of descriptive information (metadata) for data, services, and related information objects.
```

Comment:
What is `full fidelity` mean?

Reply:
legacy content - from context, I'd say it means completed implementing the profile. Can be removed if it causes confusion